SPIE2016-9913-16/9913-16.tex

```diff
@@ -0,0 +1,303 @@
+\end{figure}
+
+The software is spread among many different root packages within the \texttt{lsst} namespace:
+
+\begin{description}
+\item[\textbf{afw}] These provide core LSST functionality including geometric shapes, images, source detection, convolution and high-level exposure objects.
+
+\item[\textbf{daf}] The data access framework, also known as \emph{the Butler}, provides an abstraction layer hiding data formats and locations from the user and providing them with standardized Python objects representing the data.
+
+\item[\textbf{ip}] Image processing algorithms such as instrument signature removal and image differencing.
+
+\item[\textbf{meas}] Measurement algorithms.
+
+\item[\textbf{obs}] Observatory plugins to teach the system how to handle files from different instruments.
+
+\item[\textbf{pipe}] Pipeline processing infrastructure. This allows discrete units of computation to be linked together to form pipelines.
+
+\end{description}
+
+In terms of overlap with Astropy and affiliate packages the AFW and \texttt{meas} packages are most relevant for immediate investigation.
+
+
+\section{Astropy}
+
+Python has become the de facto standard programming language for astronomy\cite{2000ASPC..216...59G,2006ASPC..351..343H,2006ASPC..351..497K,2011ASPC..442..425G,2012SPIE.8451E..02J} and is now the default choice for new projects and for teaching.
+The Astropy\cite{2013A&A...558A..33A} project grew out of a support mailing list in 2011 to become a very successful hub for the development of Python packages for astronomy.
+At the time of writing Astropy consists of approximately 105,000 lines of Python code.
+Astropy has two key approaches to drive adoption.
+The first is to place generic astronomy code into the \emph{core} package that is brought in via \texttt{import astropy} to provide a solid, well tested, base for astronomy software development; the second is to support and encourage the concept of \emph{affiliate} packages that are packages that use the \texttt{astropy} package and follow the same philosophy on distribution and openness.
+Together these aim to provide a \emph{one stop shop} for people trying to use Python in their astronomical research.
+
+\section{Synergies}
+
+The Astropy and LSST software projects are both attempting to write astronomical software in Python that will benefit the community.
+In LSST's case we are constrained by the requirements of MREFC funding to deliver a data management system on schedule and on budget and we can justify involvement with Astropy if it allows us to deliver the functionality without impacting schedule, or if opening up the LSST algorithms to the Astropy community encourages early adoption (through ease of installation) which can lead to more extensive testing.
```

Comment:
The entire section is meant to be making the case of why Astropy would be interested. I separated out things that Astropy and LSST can collaborate on from things that LSST should do to enhance interoperability. I'll have a think about adding a sentence here to make that clear.